src/Eagle.ts

```diff
@@ -2640,11 +2646,25 @@ export class Eagle {
         if (triggerClass){
             //this is for setting toggle icons in the translation menu, as the collapse functions differently and the content is nested differently.
             //the class "closedIcon" turns the collapse arrow icon by 270 degrees and is being toggled depending on the current state of the collapse.
-            $(".translationToggle").addClass("closedIcon")
+            $(".translationToggle").addClass("closedIcon");
             var toggleState = collapseTarget.parent().parent().parent().children(".collapse").hasClass('show');
```

Comment:
I think collapseTarget is the "i" tag (the icon), so that might be a better name for the collapseTarget variable.

Here we use 3 parent() calls, lower down we use 2 parent() calls. It would be nice if we could standardise the arrangement of the HTML, so that the collapseTarget and the collapsable object always have the same relative arrangement.